hax/hax.ini

```diff
@@ -135,6 +135,12 @@ corrections_definitions = {
         {"run_min": 10223, "run_max": 12089, "correction": "FDC_SR1_data_driven_time_dependent_3d_correction_part3_v1.json.gz"},
         {"run_min": 12090, "correction": "FDC_SR1_data_driven_time_dependent_3d_correction_part4_v1.json.gz"}
     ],
+    "fdc_3d_tfnn": [
+        {"run_min": 6386, "run_max": 8648, "correction": "FDC_SR1_data_driven_time_dependent_3d_correction_tf_nn_part1_v1.json.gz"},
+        {"run_min": 8649, "run_max": 10976, "correction": "FDC_SR1_data_driven_time_dependent_3d_correction_tf_nn_part2_v1.json.gz"},
+        {"run_min": 10977, "run_max": 13195, "correction": "FDC_SR1_data_driven_time_dependent_3d_correction_tf_nn_part3_v1.json.gz"},
```

Comment:
@jingqiangye Why does this have different run binning than FANN 3D FDC?

Reply:
Because TFNN was using more Kr data for FDC. FANN 3D FDC was using Kr data up to Oct 2017, and TFNN is to Nov 2017. FANN 3D FDC will be updated with latest Kr(Jan. 2, 2018)today.